src/features/list-view/ListView.js

```diff
@@ -22,33 +22,44 @@ type Props = {
     getGridHeader: (columnIndex: number) => any,
     getGridHeaderSort?: (columnIndex: number) => typeof SORT_ORDER_ASCENDING | typeof SORT_ORDER_DESCENDING | null,
     height: number,
+    isGridHeaderSortable?: (columnIndex: number) => boolean,
     onSortChange?: (columnIndex: number) => void,
     rowCount: number,
     width: number,
 };
 
 class ListView extends React.PureComponent<Props> {
     cellRenderer = ({ columnIndex, key, rowIndex, style }: CellRendererArgs) => {
-        const { getGridCell, getGridHeader, getGridHeaderSort = noop, onSortChange = noop } = this.props;
+        const {
+            getGridCell,
+            getGridHeader,
+            getGridHeaderSort = noop,
+            isGridHeaderSortable = noop,
+            onSortChange = noop,
+        } = this.props;
 
         if (rowIndex === 0) {
             const displayName = getGridHeader(columnIndex);
             const sortDirection = getGridHeaderSort(columnIndex);
             const isSortAsc = sortDirection === SORT_ORDER_ASCENDING;
-            const className = classNames({
+            const iconClassName = classNames({
                 'bdl-ListView-isSortAsc': isSortAsc,
             });
+            const isHeaderSortable = isGridHeaderSortable(columnIndex);
 
+            const headerClassName = classNames('bdl-ListView-columnHeader', {
```

Comment:
headerClassName seems generic, not descriptive enough.
Perhaps we could change this to something else?

Reply:
There's only one header within this ListView component so I think it's appropriate that we have a generic header class name. If there were multiple headers in this component then I would push for a more specific name.